src/Microsoft.Win32.Primitives/ref/Microsoft.Win32.Primitives.cs

```diff
@@ -8,13 +8,15 @@
 
 namespace System.ComponentModel
 {
-    public partial class Win32Exception : System.Exception
+    public partial class Win32Exception : System.Runtime.InteropServices.ExternalException
```

Comment:
Yes please, otherwise it will show up as a spurious diff and we want to drive the list to zero without noise.